pkgs/top-level/all-packages.nix

```diff
@@ -10708,6 +10708,10 @@ with pkgs;
     '';
   };
 
+  libxml2Static = appendToName "static" (libxml2.override {
```

Comment:
I would prefer to leave this out of all-packages.nix and just have people override it in their own configs.

Reply:
Indeed, we typically build only a single version of a package.